minio/minio
# Corrected the count of buckets and objects graphs

## Contribution License
All contributions in this pull request are licensed to the project maintainers 
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
In distributed setup with a load balancer, randomly any server would report the metrics `minio_cluster_bucket_total` and `minio_cluster_usage_object_total` and while graphing it, we should take max of reported values.

## Motivation and Context


## How to test this PR?
Load the dashboard in grafana with a distributed MinIO deployment with Nginx or Sidekick configured. Use LB URL to setup prometheus scraping and make sure both `Number of Buckets` and `Number of Objects` values are shown properly.

## Types of changes

## Checklist:
